tst/run_tests.py

```diff
@@ -221,6 +221,11 @@ def main(**kwargs):
         "Summary: {0} out of {1} {2} "
         "passed\n".format(num_passed, num_tests, test_string)
     )
+
+    # Erase simulation output data unless requested that it be saved
+    if kwargs.pop("erase_data"):
+        os.system(f"rm -rf {artemis.artemis_data_dir}")
```

Comment:
open to changing to `artemis.get_artemis_dir()` for consistency elsewhere?

Reply:
I wanted to leave stuff like figures and logs, only erase the `*.phdf` outputs from artemis that get written to `artemis_data_dir`. So I think this naming is OK as is?

Reply:
Ah sorry... I am still waking up this morning... 

I meant using `artemis.get_data_dir()`

i.e., just using the getter function

Reply:
Ah yes you're right, I made that switch to the getter function, thanks